### ISSUE TITLE
Addition of Database

### ISSUE THREAD

**Issue by mayank-0407** [kabaddiPy/kabaddiPy#20]
Create a PostgreSQL database schema using Prisma to store all the data currently in JSON files, optimizing storage for the Python package. After creating the schema, integrate it into the models so they fetch data from the database instead of the JSON files.

**Comment by catalystaviral** [comment]
hi just wanna know if this issue is still open to contribute 
will i get both point for gssoc and badge for hacktober ???
[edit: how much points will i get]

**Comment by mayank-0407** [comment]
@COPYRIGHTXO
Yes, this issue is open. It will be considered for GSSOC as the 'gssoc-ext' tag has been added. For Hacktoberfest, they pull PRs from GitHub, so it will be considered there as well.

**Comment by mayank-0407** [comment]
@COPYRIGHTXO It will be a level2 contribution

**Comment by catalystaviral** [comment]
Sir so am I allowed to work on this??!!....

**Comment by mayank-0407** [comment]
Yes

**Comment by catalystaviral** [comment]
Appreciate that !!...

**Comment by catalystaviral** [comment]
Sir @mayank-0407,
I'm having issues with my pc isn't supporting this back end thing can you please assign me for sometime related to front end because it will work in my pc

**Comment by mayank-0407** [comment]
@COPYRIGHTXO You will need to install Python and install requirements to run the project here we do not have a frontend as this project is all about the development of the Python library.
